app/Http/Controllers/MovieController.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Movie;
+use Illuminate\Http\Request;
+use App\Http\Requests\MovieRequest;
+
+class MovieController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return Movie::all();
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(MovieRequest $request)
+    {
+        return Movie::create($request->only(['title','description','imageCover','genre']));
+    }
+
+    /**
+     * Display the specified resource.
+     *
+     * @param  \App\Models\Movie  $movie
+     * @return \Illuminate\Http\Response
+     */
+    public function show(MovieRequest $movie)
+    {
+        return $movie;
+    }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param  \App\Models\Movie  $movie
+     * @return \Illuminate\Http\Response
+     */
+    public function edit(Movie $movie)
+    {
+        //
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @param  \App\Models\Movie  $movie
+     * @return \Illuminate\Http\Response
+     */
+    public function update(MovieRequest $request, Movie $movie)
+    {
+        return $movie->update($request->only(['title','description','imageCover','genre'])) ? $movie : [];
```

Comment:
$this->imaService->updateMove($request->only(['title','description','imageCover','genre']));